README.md

```diff
@@ -105,6 +105,16 @@ in-toto-verify --layout <layout path>
                --layout-keys (<layout pubkey path>,...)
 ```
 
+
+#### Settings
+in-toto provides a settings module as a place to store defaults that can
+be used throughout the package. When using in-toto from the command line,
+these settings can be configured using environment variables or RCfiles.
+Take a look at [`in_toto.settings`](https://github.com/in-toto/in-toto/blob/develop/in_toto/settings.py)
+to learn more about the available defaults and at
+[`in_toto.user_settings`](https://github.com/in-toto/in-toto/blob/develop/in_toto/user_settings.py)
```

Comment:
I get a 404 error with this link.

Reply:
I'm not so sure about the  documentation of this section.  Is the command-line tool intended to be run by a variety of users?  If so, wouldn't linking to a Python module might be inappropriate for these users?

Reply:
Re 404: should have used relative path (`user_settings.py` isn't in the main branch yet).
Re inappropriate: Agreed, will change.